projects/i18n/src/lib/pipe/display-translation.pipe.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { AsyncPipe } from '@angular/common';
+import { ChangeDetectorRef, Injectable, Pipe, PipeTransform } from '@angular/core';
+import { TranslatedText, TranslationService } from '../service/translation-service';
+
+/**
+ * Displays the given {@link TranslatedText} to the page.
+ */
+@Injectable()
+@Pipe({
+    name: 'displayTranslation',
+    pure: false,
+})
+export class DisplayTranslationPipe implements PipeTransform {
+    constructor(private changeDetector: ChangeDetectorRef) {}
+
+    transform(value: TranslatedText): any {
+        if (typeof value === 'string') {
```

Comment:
I think this is simple enough for a ternary
```ts
return typeof value === 'string' 
    ? value 
    : new AsyncPipe(this.changeDetector).transform(value)
```
If you don't love that, we avoid elses after return statements.

```ts
if (typeof value === 'string') {
    return value;
}
return new AsyncPipe(this.changeDetector).transform(value);
```